langkit/ocaml_api.py

```diff
@@ -253,6 +253,35 @@ def wrap_requires_context(self, type):
             ' (wrapping): {}'.format(type)
         ))
 
+    def have_ctype_view(self, type):
+        """
+        Given a type, return true if it has a ctype view. Having a ctype view
+        means that wrap and unwrap operations are not needed explicitely.
+
+        :param ct.CompiledType type: Type we want to check for having a view
+            or not.
```

Comment:
“The type to check.”?

Reply:
Maybe my sentence was a bit to complex, agreed :p